src/js/controllers/tab-send.controller.js

```diff
@@ -61,6 +61,49 @@ angular.module('copayApp.controllers').controller('tabSendController', function(
     });
   });
 
+  $scope.closePopover = function() {
+    $scope.popover.hide();
+  };
+
+  $scope.$on('$destroy', function() {
+    $scope.popover.remove();
+  });
+
+  $scope.confirmOpenapAddress = function(address, coin){
+    $scope.popover.remove();
+    var params = sendFlowService.state.getClone();
+    params.data = address;
+    params.coin = coin;
+    sendFlowService.start(params, function onError() {
+      return
+    });
+  }
+  
+  $scope.resolveOpencapAlias = function(alias){
+    opencapService.getAddress(alias)
+    .then(result => {
```

Comment:
Now :  `=>`  : lambda expression is not supported for old javascript engine.
Should be : 
```
function onThen() {
}
```